plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -1357,34 +1357,34 @@ public void testEmptyTimestampClusteringKey()
 
-            assertContainsEventually(() -> computeActual("SHOW TABLES FROM cassandra." + keyspaceName), resultBuilder(getSession(), createUnboundedVarcharType())
-                    .row("table_set")
-                    .row("table_list")
-                    .row("table_map")
+            assertContainsEventually(() -> computeActual("SHOW TABLES FROM cassandra.tpch"), resultBuilder(getSession(), createUnboundedVarcharType())
+                    .row(tableNameForSet)
+                    .row(tableNameForList)
+                    .row(tableNameForMap)
                     .build(), new Duration(1, MINUTES));
 
-            session.execute("INSERT INTO " + keyspaceName + ".table_set (column_5, nested_collection) VALUES (1, {{1, 2, 3}})");
-            assertThat(computeActual("SELECT nested_collection FROM cassandra." + keyspaceName + ".table_set").getMaterializedRows().get(0)).isEqualTo(new MaterializedRow(DEFAULT_PRECISION, "[[1,2,3]]"));
+            session.execute("INSERT INTO tpch." + tableNameForSet + " (column_5, nested_collection) VALUES (1, {{1, 2, 3}})");
+            assertThat(computeActual("SELECT nested_collection FROM cassandra.tpch." + tableNameForSet).getMaterializedRows().get(0)).isEqualTo(new MaterializedRow(DEFAULT_PRECISION, "[[1,2,3]]"));
 
-            session.execute("INSERT INTO " + keyspaceName + ".table_list (column_5, nested_collection) VALUES (1, [[4, 5, 6]])");
-            assertThat(computeActual("SELECT nested_collection FROM cassandra." + keyspaceName + ".table_list").getMaterializedRows().get(0)).isEqualTo(new MaterializedRow(DEFAULT_PRECISION, "[[4,5,6]]"));
+            session.execute("INSERT INTO tpch." + tableNameForList + " (column_5, nested_collection) VALUES (1, [[4, 5, 6]])");
+            assertThat(computeActual("SELECT nested_collection FROM cassandra.tpch." + tableNameForList).getMaterializedRows().get(0)).isEqualTo(new MaterializedRow(DEFAULT_PRECISION, "[[4,5,6]]"));
 
-            session.execute("INSERT INTO " + keyspaceName + ".table_map (column_5, nested_collection) VALUES (1, {7:{8:9}})");
-            assertThat(computeActual("SELECT nested_collection FROM cassandra." + keyspaceName + ".table_map").getMaterializedRows().get(0)).isEqualTo(new MaterializedRow(DEFAULT_PRECISION, "{7:{8:9}}"));
+            session.execute("INSERT INTO tpch." + tableNameForMap + " (column_5, nested_collection) VALUES (1, {7:{8:9}})");
+            assertThat(computeActual("SELECT nested_collection FROM cassandra.tpch." + tableNameForMap).getMaterializedRows().get(0)).isEqualTo(new MaterializedRow(DEFAULT_PRECISION, "{7:{8:9}}"));
         }
         finally {
-            dropKeyspace(keyspaceName);
+            assertUpdate("DROP TABLE IF EXISTS tpch." + tableNameForSet);
+            assertUpdate("DROP TABLE IF EXISTS tpch." + tableNameForList);
+            assertUpdate("DROP TABLE IF EXISTS tpch." + tableNameForMap);
```

Comment:
Please drop tables with `session.execute` or `onCassandra`. We use the same SQL executor between CREATE and DROP in tests. Also, using a Trino executor is troublesome in this connector because there is a delay until a new object is visible.